build_library/build_image_util.sh

```diff
@@ -294,33 +294,57 @@ finish_image() {
   # Zero all fs free space to make it more compressible so auto-update
   # payloads become smaller, not fatal since it won't work on linux < 3.2
   sudo fstrim "${root_fs_dir}" || true
-  if mountpoint -q "${root_fs_dir}/usr"; then
```

Comment:
and since a lot of the following shouldn't happen without a /usr partition it may need to move to a new function to keep things clean, nesting things under a big if could work too, whatever looks better.

the only time there isn't a /usr partition is when running `./build_image container` which creates a dev image with a single partition, useful for passing to nspawn (not useful for booting)